Sources/StreamChat/Config/ChatClientConfig.swift

```diff
@@ -91,15 +91,20 @@ public struct ChatClientConfig {
     /// - Parameter apiKey: The API key of the chat app the `ChatClient` connects to.
     ///
     
-    /// Allows to inject a custom API client for uploading attachments, if not specified `StreamCDNClient` is used
-    public var customCDNClient: CDNClient?
+    public enum CDNClientConfig {
```

Comment:
Would be great to have some docs here 👍

Reply:
It seems like we might want to drop this change in the end if the customer will be able to achieve e2e without it